### PR TITLE
Integrate Activation+Mail API 2.1.0-RC2 + Angus-Activation+Mail 1.0.0-M1

### PR DESCRIPTION
**Pull request by lukasj**
QL passed on my end. Clean up will have to be done once there is full Metro available

Note that this may conflict with #23694

### CODE REVIEW

File: appserver/featuresets/glassfish/pom.xml [hunk 1]
```diff
@@ -795,8 +795,18 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>com.sun.activation</groupId>
-            <artifactId>jakarta.activation</artifactId>
+            <groupId>jakarta.activation</groupId>
+            <artifactId>jakarta.activation-api</artifactId>
+            <exclusions>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply appserver/featuresets/glassfish/pom.xml:800]: No. At the featureset level, one needs full control over artifacts while at the regular project level, one prefers availability of transitive dependencies. Having transitive deps available is also useful when creating classpaths for manifests etc

**Comment by lukasj** [comment]
second commit (integration of mail) also adds activation to places where I believe it was missing (ie to specs in pom, default osgi imports)

**Comment by arjantijms** [comment]
Note that these are the dependencies for Jakarta EE 10, so that means we need to create the 6.x branch before merging this.

**Comment by arjantijms** [comment]
The 6.x branch has been created. Master will transition to GF 7 / EE 10.

**Review by arjantijms — APPROVED** [review]

**Comment by dmatej** [comment]
Just a not for myself - commit: 59981856a4c8591b58f2cdc523311c4ab8d08595

**Pull request merged**